plasmapy/examples/langmuir_analysis.py

```diff
@@ -0,0 +1,89 @@
+# coding: utf-8
+"""
+Analyzing Langmuir
+=========================
+
+Let's analyze various Langmuir probe characteristics using the
+`diagnostics.langmuir` subpackage.
+"""
+
+import astropy.units as u
+from plasmapy.diagnostics.langmuir import Characteristic, swept_probe_analysis
+import pickle
+import numpy as np
+
+######################################################
+"""The first characteristic we analyze is a simple single-probe measurement in
+a low (ion) temperature, low density plasma with a cylindrical probe. This
+allows us to utilize OML theory implemented in `swept_probe_analysis`.
+The data has been preprocessed with some smoothing, which allows us to obtain
+a nice Electron Energy Distribution Function (EEDF) as well.
+"""
+
+# Load the bias and current values stored in the .p pickle file.
+[bias, current] = pickle.load(open('langmuir_samples\\Beckers2017.p', 'rb'))
+
+# Create the Characteristic object, taking into account the correct units
+characterstic = Characteristic(np.array(bias) * u.V,
+                               np.array(current)*1e3 * u.mA)
+
+# Calculate the cylindrical probe surface area
+probe_l = 1.145 * u.mm
+probe_d = 1.57 * u.mm
+probe_area = probe_l * np.pi * probe_d + np.pi * 0.25 * probe_d**2
+
+# Perform the analysis. Since the plasma is in Helium a ion mass number of 4
+# is entered. The results are visualized and the obtained EEDF is also shown.
+print(swept_probe_analysis(characterstic, probe_area, 4,
+                           visualize=True, plot_EEDF=True))
```

Comment:
Once you start using newlines to separate arguments, I would just use newlines throughout instead of mixing horizontal and vertical spacing.